packages/scxa-cell-type-wheel-experiment-heatmap/cypress/component/CellTypeWheelExperimentHeatmap.cy.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+
+import CellTypeWheelExperimentHeatmap from '../../src'
+import speciesResponse from '../fixtures/speciesRespone.json'
+import cellTypeWheelData from '../fixtures/CellTypeDataForWheelResponse.json'
+import heatMapTCellResponse from '../fixtures/heatMapRegulatoryTCellResponse.json'
+
+describe(`CellTypeWheelExperimentHeatmap`, () => {
+  const props = {
+    searchTerm: `t cell`
+  }
+
+  beforeEach(() => {
+    cy.intercept(`GET`, `/gxa/sc/json/cell-type-wheel/t%20cell`,
+      cellTypeWheelData)
+  })
+
+  it(`mounts without data and without species provider`, () => {
+    cy.viewport(800, 1000)
+    cy.mount(<CellTypeWheelExperimentHeatmap
+      {...props}
+    />)
+    cy.contains(`Failed fetching species`)
+  })
+
+  it(`mounts without data`, () => {
+    cy.intercept(`GET`, `/gxa/sc/json/suggestions/species`,
+      speciesResponse)
+    cy.viewport(800, 1000)
+    cy.mount(<CellTypeWheelExperimentHeatmap
+      {...props}
+    />)
+  })
+
+  it(`mounts with data`, () => {
```

Comment:
You’re doing a bit more than just mounting the component. I think the name of the test should reflect this, something like “displays a heatmap when a cell type label is clicked” or something like that.

Reply:
ah, you are right, I forgot to update the title of this test

Reply:
Changed the title of this test.